ui/src/adapters/api/credentials.ts

```diff
@@ -87,12 +161,95 @@ export const credentialStatusParser = getStrictParser<CredentialStatus>()(
 export async function getCredential({
   credentialID,
   env,
+  issuerIdentifier,
   signal,
 }: {
   credentialID: string;
   env: Env;
+  issuerIdentifier: IssuerIdentifier;
   signal?: AbortSignal;
 }): Promise<Response<Credential>> {
+  try {
+    const credentialDetail = await getCredentialDetail({
+      credentialID,
+      env,
+      issuerIdentifier,
+      signal,
+    });
+
+    if (!credentialDetail.success) {
+      return credentialDetail;
+    }
+
+    const {
+      "@context": context,
+      credentialSchema,
+      credentialStatus,
+      credentialSubject,
+      expirationDate,
+      id,
+      issuanceDate,
+      issuer,
+      proof,
+    } = credentialDetail.data;
+
+    const revocationStatus = await getRevocationStatus({
+      env,
+      issuerIdentifier,
+      nonce: credentialStatus.revocationNonce,
+      signal,
+    });
+
+    if (!revocationStatus.success) {
+      return revocationStatus;
+    }
+
+    const proofTypes = proof.map(({ type }) => type);
+
+    const schemaHash = getSchemaHash({
+      id: `${context.at(-1)}#${credentialSubject.type}`,
```

Comment:
@vmidyllic please review this line, because it looks incorrect to me.

Reply:
Agreed, this should fail for schemas from the schema builder since its ids are a urn:uuid,

`"@id": "urn:uuid:9bbc4cd1-8658-4a48-b013-87603e32e399"`

Ids like this one below are the old ones:

`"@id": "https://raw.githubusercontent.com/anima-protocol/claims-polygonid/main/schemas/json-ld/pol-v1.json-ld#AnimaProofOfLife"`